### PR TITLE
update default octez to v12

### PR DESCRIPTION
**Pull request by nicolasochem**
Change version in test value files as well. Also fix a few outdated tests.

### CODE REVIEW

**Review by orcutt989 — APPROVED** [review]
Awesome!

File: test/charts/mainnet.expect [hunk 1]
```diff
@@ -146,7 +151,7 @@ spec:
           readinessProbe:
             httpGet:
               path: /is_synced
-              port: 31732                                                
+              port: 31732
```
Comment (harryttd) [inline test/charts/mainnet.expect:154]: theres some random white space being added in a bunch of places 🤷

Comment (nicolasochem) [reply test/charts/mainnet.expect:154]: There is a helper script to re-generate the expected helm template. I ran it to update to v12 everywhere in the tests, then this happened. I believe whitespace was already there. Perhaps a tab vs space situation. This should be addressed separately.

Comment (elric1) [reply test/charts/mainnet.expect:154]: I think that what's going on here is that a prior commit didn't update the expect files. Once we enforce this on every pull request we won't see this.

**Pull request merged**